openlibrary/plugins/ol_infobase.py

```diff
@@ -322,7 +322,8 @@ def http_notify(site, old, new):
 
     for url in config.http_listeners:
         try:
-            response = urllib.request.urlopen(url, json).read()
+            response = requests.get(url, json_data).text
```

Comment:
```suggestion
            response = requests.get(url, params=json_data).text
```
Please name the second function parameter as is done consistently in the requests docs.  It is better to be explicit here because params=, data=, json= are all possible and we do not want the reader to guess wrong.